### DISCUSSION TITLE
Adding more features, updating the readme and making the python api more consumable for devs

### DISCUSSION THREAD

**Discussion by mishraomp** [bcgov/quickstart-openshift#1140] [General]
@conbrad @franTarkenton @DerekRoberts  since the backend-python api is now added as a pluggable backend, I am thinking to make sure that it is easily consumable for devs, your inputs and contributions are super valuable. starting this thread to discuss and capture ideas.

**Comment by DerekRoberts** [comment]
Workable examples!  What could be better?

**Reply by conbrad** [reply]
Agreed, also maybe a small cli to generate your API framework of choice?

**Reply by DerekRoberts** [reply]
^ I like that idea.  We could make sure it pulls out hard coded repo links and other tough-to-template headaches.

**Reply by mishraomp** [reply]
> ^ I like that idea. We could make sure it pulls out hard coded repo links and other tough-to-template headaches.

are you planning to hammer that down, I will be adding (java and golang soon) ;)

**Reply by DerekRoberts** [reply]
No plans, but we'll have to figure something out!  How's this for a start?
https://github.com/bcgov/quickstart-openshift/issues/1141

**Comment by mishraomp** [comment]
closing this as no longer relevant
